Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -170,6 +179,33 @@ def get_id_offset():
     )
 
 
+def get_custom_fields():
+    """
+    This function returns all custom fields.
+    :return: dict of custom fields: id as key and description as value.
+    """
+    custom_id_description_mapping = {}
+    HEADERS['Accept'] = "application/json"
+    try:
+        res = requests.request(
```

Comment:
why dont you use jira_req()?

Reply:
Because I want a different behavior: it uses return_error which exits the function and I can't use "try catch".